tests/src/test/java/org/wso2/micro/gateway/tests/common/MockHttpServer.java

```diff
@@ -182,6 +188,29 @@ public void handle(HttpExchange exchange) throws IOException {
                     exchange.close();
                 }
             });
+            httpServer.createContext(TMRestAPIBasePath + "/throttleEventReceiver", new HttpHandler() {
+                public void handle(HttpExchange exchange) throws IOException {
+                    String jsonRequest = IOUtils.toString(exchange.getRequestBody());
+                    if (count == 9 || count == 19 || count == 29 || count == 39 || count == 49) {
+                        JsonParser jsonParser = new JsonParser();
+                        JsonObject jsonObject = (JsonObject) jsonParser.parse(jsonRequest);
+                        JMSPublisher jmsPublisher = new JMSPublisher();
+
+                        try {
+                            jmsPublisher.getJson(jsonObject);
+                        } catch (JMSException e) {
```

Comment:
Better to write a specific error message instead of logging the same error message in both catch blocks.

Reply:
Fixed with de596b77c7f9217eca794e47cd29e36dbd24b0d4